feature-libs/order/components/order-details/order-details.module.ts

```diff
@@ -35,9 +41,42 @@ import { OrderDetailItemsComponent } from './order-detail-items/order-detail-ite
 import { OrderDetailReorderComponent } from './order-detail-reorder/order-detail-reorder.component';
 import { ReorderDialogComponent } from './order-detail-reorder/reorder-dialog/reorder-dialog.component';
 import { OrderDetailTotalsComponent } from './order-detail-totals/order-detail-totals.component';
+import {
+  DownloadOrderInvoicesDialogModule,
+  MyAccountV2ConsignmentTrackingComponent,
+  MyAccountV2OrderDetailsActionsComponent,
+} from './myaccount-v2';
 import { OrderOverviewComponent } from './order-overview/order-overview.component';
 import { defaultReorderLayoutConfig } from './reoder-layout.config';
 
+function registerOrderOutletFactory(): () => void {
+  const token = inject(MYACCOUNT_V2_ORDER);
```

Comment:
[NITPICK]
What about more meaningful variable name? e.g.
```suggestion
  const isMyAccountV2 = inject(MYACCOUNT_V2_ORDER);
```

Reply:
done